deepchem/models/tests/test_scscore_pytorch.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import numpy as np
+
+import unittest
+
+try:
+  import torch
+  import torch.nn as nn
+  from deepchem.models.torch_models.scscore import ScScore
+  has_torch = True
+except:
+  has_torch = False
+  
+  
+@unittest.skipIf(not has_torch, 'torch is not installed')
+@pytest.mark.torch
+def test_scscore_pytorch():
```

Comment:
A test for model should cover model overfit and model reload.

Reply:
Oh!